esp/esp/customforms/DynamicForm.py

```diff
@@ -646,14 +649,23 @@ def getResponseData(self, form):
                 # Include this field only if it isn't a dummy field
             elif generic_fields[ftype]['typeMap'] is not DummyField:
                 response_data['questions'].append([qname, field['label'], ftype])
-            
+
+        users = ESPUser.objects.filter(id__in=map(lambda response: response['user_id'], responses)).distinct()
```

Comment:
I believe `ESPUser.objects.in_bulk` does the same thing as this, but this seems fine too.